tslint.json

```diff
@@ -5,6 +5,7 @@
     "no-any": false,
     "quotemark": [true, "single", "avoid-escape"],
     "member-ordering": false,
-    "no-string-based-set-timeout": false
+    "no-string-based-set-timeout": false,
```

Comment:
Basically without this, there is a conflict between prettier and tslint.
Prettier will format and add the semicolon (and if you have it set to format on save in VS Code, this does it automatically).
But, tslint, will complain that the semicolon is unnecessary.

This change fixes it so that tslint wil allow that semicolon and play nice with prettier.

I'm opening a PR to accept this in @salesforce/dev-config but for now we can address it locally.